devsetup/tripleo/tripleo_install.sh

```diff
@@ -15,11 +15,32 @@
 # under the License.
 set -ex
 
```

Comment:
@bogdando please check slack when you get some time

this breaks the ceph job (your test ran multinode-no-ceph) so please have a look and if we cannot find a good solution today then we can revert and work it out before going again